### PR TITLE
When releasing CDI, cross compile arm64 as well

### PR DESCRIPTION
**Pull request by awels**
We have been releasing periodic arm64 containers for some time now. We should make an official release as well.

### CODE REVIEW

**Review by brianmcarey — COMMENTED** [review]
It might be better to move this job to the workloads cluster `kubevirt-prow-workloads` and increase the timeout - I think the cross compile will add a lot of run time and require more performance.

**Comment by awels** [comment]
This is doing essentially what is happening in the periodic and that is running in the `kubevirt-prow-workload` cluster, but the timeout is the same, I can increase it because I am running 2 builds.

**Comment by nixpanic** [comment]
You probably want to correct the subject of the PR, `s/amd64/arm64/`. Increasing the timeout for cross-compiling is probably a good idea indeed. I'd :+1:  the changes, but it won't count for getting this merged.

File: github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml [hunk 1]
```diff
@@ -29,9 +31,10 @@ postsubmits:
         - "-c"
         - |
           cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io &&
-          # Only push images on tags
+          # Only push images on tags, for amd64 and aarch64 (crossbuild, since we have no aarch64 builder)
           [ -z "$(git tag --points-at HEAD | head -1)" ] ||
-          DOCKER_TAG="$(git tag --points-at HEAD | head -1)" make bazel-push-images
+          DOCKER_TAG="$(git tag --points-at HEAD | head -1)-amd64" make bazel-push-images ||
```
Comment (akalenyu) [inline github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml:36]: Reassigning `DOCKER_TAG`? Is this what you meant?

Comment (awels) [reply github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml:36]: yes if you look at what kubevirt does here, they do the same, making a tag for amd and arm and quay seems to correctly re-arrange that.

File: github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml [hunk 2]
```diff
@@ -29,9 +31,10 @@ postsubmits:
...
+          DOCKER_TAG="$(git tag --points-at HEAD | head -1)-arm64" BUILD_ARCH="crossbuild-aarch64" make bazel-push-images
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml:37]: Would you also want to create a combined manifest? 

something like:
```
podman manifest create --amend $IMAGE_NAME $IMAGE_NAME-amd64 $IMAGE_NAME-arm64
podman manifest push $IMAGE_NAME
```

**Comment by awels** [comment]
/hold
I will likely close this in favor of a script in the CDI repo.

File: github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml (hunk 2 above)
Comment (brianmcarey) [reply github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml:37]: Sounds good to me

**Pull request closed without merging**